book/src/dev/zebra-dependencies-for-audit.md

```diff
@@ -0,0 +1,150 @@
+| [rand_core](https://github.com/rust-random/rand) | [0.6.4](https://github.com/rust-random/rand/releases/tag/0.6.4) | no audits, but seems widely used
+| [sha2](https://github.com/RustCrypto/hashes) | 0.9.9 | no audits, but seems widely used
+| [ripemd](https://github.com/RustCrypto/hashes) | 0.1.3 | no audits, but seems widely used
+| [secp256k1](https://github.com/rust-bitcoin/rust-secp256k1/) | 0.21.3 | no audits, but seems widely used
+| [subtle](https://github.com/dalek-cryptography/subtle) | [2.4.1](https://github.com/dalek-cryptography/subtle/releases/tag/2.4.1) | no audits, but seems widely used
+| [fpe](https://github.com/str4d/fpe) | 0.5.1 |  I think it's not being used yet
+| [group](https://github.com/zkcrypto/group) | [0.12.0](https://github.com/zkcrypto/group/releases/tag/0.12.0) | no audits but it's just traits, seems widely used
+| [x25519-dalek](https://github.com/dalek-cryptography/x25519-dalek) | [1.2.0](https://github.com/dalek-cryptography/x25519-dalek/releases/tag/1.2.0) | no audits, but seems widely used
+| [jubjub](https://github.com/zkcrypto/jubjub) | [0.9.0](https://github.com/zkcrypto/jubjub/releases/tag/0.9.0) | not sure if were covered by ECC audits. Seem widely used.
+| [bellman](https://github.com/zkcrypto/bellman) | 0.13.1 | not sure if were covered by ECC audits. Seem widely used.
+
+## Async code and services
+
+| Name | Version | Notes
+|------| ------- | -----
+| [futures](https://github.com/rust-lang/futures-rs) | [0.3.24](https://github.com/rust-lang/futures-rs/releases/tag/0.3.24) | 
+| [futures-core](https://github.com/rust-lang/futures-rs) | [0.3.24](https://github.com/rust-lang/futures-rs/releases/tag/0.3.24) |
+| [pin-project](https://github.com/taiki-e/pin-project) | [1.0.12](https://github.com/taiki-e/pin-project/releases/tag/v1.0.12) | 
+| [rayon](https://github.com/rayon-rs/rayon) | [1.5.3](https://github.com/rayon-rs/rayon/releases/tag/v1.5.3) |
+| [tokio](https://github.com/tokio-rs/tokio) | 1.21.2 |
+| [tokio-util](https://github.com/tokio-rs/tokio) | 0.7.4 | 
+| [tower](https://github.com/tower-rs/tower) | 0.4.13 |
+| [futures-util](https://github.com/rust-lang/futures-rs) | [0.3.24](https://github.com/rust-lang/futures-rs/releases/tag/0.3.24) |
+| [tokio-stream](https://github.com/tokio-rs/tokio) | 0.1.10 |
+| [hyper](https://github.com/hyperium/hyper) | [0.14.20](https://github.com/hyperium/hyper/releases/tag/v0.14.20) |
+| [jsonrpc-core](https://github.com/paritytech/jsonrpc) | [18.0.0](https://github.com/paritytech/jsonrpc/releases/tag/v18.0.0) | 
+| jsonrpc-derive | 18.0.0 
+| [jsonrpc-http-server](https://github.com/paritytech/jsonrpc) | [18.0.0](https://github.com/paritytech/jsonrpc/releases/tag/v18.0.0) | 
+
+## Types and encoding
+
+| Name | Version | Notes
+|------| ------- | -----
+| [bitflags](https://github.com/bitflags/bitflags) | [1.3.2](https://github.com/bitflags/bitflags/releases/tag/1.3.2) 
```

Comment:
This crate is consensus rules for orchard transaction fields however the crate has [1,776](https://crates.io/crates/bitflags/reverse_dependencies) including a bunch of important ones. I think we should not audit.

Reply:
We want to avoid auditing code we didn't write.